packages/graphback-codegen-schema/src/definitions/schemaDefinitions.ts

```diff
@@ -258,8 +281,64 @@ export const buildMutationInputType = (modelType: GraphQLObjectType) => {
         return fieldObj;
       }, {})
   });
+
+  schemaComposer.add(mutationInputObject)
+}
+
+function mapObjectInputFields(schemaComposer: SchemaComposer<any>, fields: GraphQLField<any, any>[]): GraphQLInputField[] {
+  return fields.map((field: GraphQLField<any, any>) => {
+    let namedType = getNamedType(field.type)
+    let typeName = namedType.name
+
+    let inputType
+    if (isObjectType(namedType)) {
```

Comment:
What happens if the type is a model?

Reply:
An unhandled error is thrown. Thanks.

Reply:
Great. This works fine but it will fail when the user adds the `SchemaPlugin` to print out the schema 

```
    Error: Non-model NoteResultList.items maps to Graphback model (Note) which is not allowed.
        at /home/manyanda/Documents/Projects/aerogear/graphback/packages/graphback-codegen-schema/src/definitions/schemaDefinitions.ts:296:15
        at Array.map (<anonymous>)
        at mapObjectInputFields (/home/manyanda/Documents/Projects/aerogear/graphback/packages/graphback-codegen-schema/src/definitions/schemaDefinitions.ts:289:17)
        at Object.addCreateObjectInputType (/home/manyanda/Documents/Projects/aerogear/graphback/packages/graphback-codegen-schema/src/definitions/schemaDefinitions.ts:319:13)
        at /home/manyanda/Documents/Projects/aerogear/graphback/packages/graphback-codegen-schema/src/SchemaCRUDPlugin.ts:731:11
        at Map.forEach (<anonymous>)
        at SchemaComposer.forEach (/home/manyanda/Documents/Projects/aerogear/graphback/node_modules/graphql-compose/lib/TypeStorage.js:37:23)
        at SchemaCRUDPlugin.createSchemaCRUDTypes (/home/manyanda/Documents/Projects/aerogear/graphback/packages/graphback-codegen-schema/src/SchemaCRUDPlugin.ts:724:20)
        at SchemaCRUDPlugin.buildSchemaForModels (/home/manyanda/Documents/Projects/aerogear/graphback/packages/graphback-codegen-schema/src/SchemaCRUDPlugin.ts:135:10)
        at SchemaCRUDPlugin.transformSchema (/home/manyanda/Documents/Projects/aerogear/graphback/packages/graphback-codegen-schema/src/SchemaCRUDPlugin.ts:63:10)
        at GraphbackPluginEngine.createSchema (/home/manyanda/Documents/Projects/aerogear/graphback/packages/graphback-core/src/plugin/GraphbackPluginEngine.ts:66:32)
        at GraphbackPluginEngine.createResources (/home/manyanda/Documents/Projects/aerogear/graphback/packages/graphback-core/src/plugin/GraphbackPluginEngine.ts:50:10)
        at Object.buildGraphbackAPI (/home/manyanda/Documents/Projects/aerogear/graphback/packages/graphback/src/buildGraphbackAPI.ts:92:33)
        at /home/manyanda/Documents/Projects/aerogear/graphback/integration/tests/runtime-workflow-mongo.ts:42:20
        at Generator.next (<anonymous>)
        at fulfilled (/home/manyanda/Documents/Projects/aerogear/graphback/node_modules/tslib/tslib.js:112:62)
        at processTicksAndRejections (internal/process/task_queues.js:97:5)

      at tests/runtime-workflow-mongo.ts:59:13

```

I ran into this issue locally when executing our integration tests - where we add the schema plugin to print out the schema so that it gets picked out by gql client
https://github.com/aerogear/graphback/blob/master/integration/tests/runtime-workflow-postgres.ts#L48-L50

But we already have a schema plugin defined by default https://github.com/aerogear/graphback/blob/master/packages/graphback/src/buildGraphbackAPI.ts#L82

/cc @wtrocki we kind of chatted about it today.